redistimeseries/client.py

```diff
@@ -362,18 +372,24 @@ def deleterule(self, source_key, dest_key):
         """Deletes a compaction rule"""
         return self.redis.execute_command(self.DELETERULE_CMD, source_key, dest_key)
 
-    def __range_params(self, key, from_time, to_time, count, aggregation_type, bucket_size_msec):
+    def __range_params(self, key, from_time, to_time, count, aggregation_type, bucket_size_msec,
+                       filter_by_ts, filter_by_value):
```

Comment:
@AvitalFineRedis I think that having `filter_by_value` as a single parameter will make it hard on users to understand the actual type/meaning. Can we split it to 2 parameters?

Reply:
sure